platform/broadcom/sonic-platform-modules-dell/z9332f/sonic_platform/sfp.py

```diff
@@ -1022,16 +1022,19 @@ def reinit_sfp_driver(self):
             #Avoid re-initialization of the QSFP/SFP optic on QSFP/SFP port.
             if (self.media_type == 'SFP' and (driver_name == 'optoe1' or driver_name == 'optoe3')):
                 subprocess.Popen(delete_device, shell=True, stdout=subprocess.PIPE)
+                time.sleep(0.2)
```

Comment:
what is the use case here? why is that pmon restart will reinit sfp driver?

Reply:
Initially, optoe3 driver will be loaded for all the ports in z9332f platform init script.
If 100G is inserted, EEPROM driver has to be changed to optoe1.
optoe1 ===> QSFP+,QSFP28
optoe2 ===> SFP+,SFP28
optoe3 ===> CMIS devices.

As optoe3 is attached for all the transceivers, pmon will reinitialize the EEPROM driver when there is a mismatch.